core/storage/job/gae_models.py

```diff
@@ -68,7 +68,8 @@ def get_new_id(cls, entity_name):
     # jobs.
     metadata = ndb.JsonProperty(indexed=False)
     # The output of the job. This is only populated if the job has status code
-    # STATUS_CODE_COMPLETED, and is None otherwise.
+    # STATUS_CODE_COMPLETED, and is None otherwise. If populated, this is
+    # expected to be a list of strings.
```

Comment:
I think this is already how output is stored, I'm just clarifying that it's a list of strings since it wasn't clear before. A repeated text property or even json property would be nicer, but that would break compatibility with previous stored jobs. :)